### ISSUE TITLE
Error when copying multiple directories to the same nonexistent directory

### ISSUE THREAD

**Issue by golopot** [vladshcherbin/rollup-plugin-copy#19]
Related issue #7 #9 .

Running rollup with the following config produces this error:
```
index.js → dist/bundle.js...
[!] (copy plugin) Error: EEXIST: file already exists, mkdir 'dist'
Error: EEXIST: file already exists, mkdir 'dist'
```

```json
{
  "rollup": "^1.11.2",
  "rollup-plugin-copy": "^2.0.0"
}
```

```js
import copy from "rollup-plugin-copy";
export default {
  input: "index.js",
  output: {
    file: "dist/bundle.js",
    format: "cjs"
  },
  plugins: [
    copy({
      targets: {
        "dir-1": "dist",
        "dir-2": "dist"
      }
    })
  ]
};
```

Edit: 
Only happens when dist/ does not exist.

**Comment by vladshcherbin** [comment]
@golopot this happens because you target same directory, the plugin copies in parallel and both processes try to create same directory if it doesn't exist.

In your example you want files from `dir-1` and `dir-2` to be in `dist` folder, right?

I can make sequential copying to prevent this error 🤔

**Comment by vladshcherbin** [comment]
Released in 2.0.1. Check, if it works for you 😉
